tools/distpartitioning/constants.py

```diff
@@ -20,9 +20,9 @@
 PART_LOCAL_NID = "part_local_nid"
 
 STR_NODE_TYPE = "node_type"
-STR_NUM_NODES_PER_CHUNK = "num_nodes_per_chunk"
+# STR_NUM_NODES_PER_CHUNK = "num_nodes_per_chunk"
```

Comment:
how about removing instead of commenting out?

Reply:
Done.